subprojects/core/src/integTest/groovy/org/gradle/api/tasks/IncrementalBuildIntegrationTest.groovy

```diff
@@ -1323,16 +1321,10 @@ task generate(type: TransformerTask) {
         '''
 
         when:
-        run 'myTask'
-        then:
-        def outputFile = file('build/output.txt')
-        outputFile.text == 'overridden private'
+        fails 'myTask'
 
-        when:
-        run 'myTask', "--info"
         then:
-        executedAndNotSkipped(":myTask")
-        outputContains("Incremental execution has been disabled to ensure correctness. Please consult deprecation warnings for more details.")
+        failure.assertThatDescription(containsNormalizedString("Type 'MyTask': property 'myPrivateInput' is private and annotated with @Input. Annotations on private getters are ignored."))
     }
 
     @ToBeImplemented("Private getters should be ignored")
```

Comment:
Same here: The `ToBeImplemented` should go away, and the test name should be changed.

It also seems like the build should fail, since the super class has a validation error, right?